Lib/xml/sax/expatreader.py

```diff
@@ -374,7 +377,7 @@ def start_element_ns(self, name, attrs):
             newattrs[apair] = value
             qnames[apair] = qname
 
-        self._cont_handler.startElementNS(pair, None,
+        self._cont_handler.startElementNS(pair, elem_qname,
```

Comment:
This will never pass `None` even with support for namespace prefixes _disabled_.

The following patch is arguably a bit silly, but it keeps things backwards compatible (and demos the idea):

```suggestion
        if not self._namespace_prefixes:
            elem_qname = None

        self._cont_handler.startElementNS(pair, elem_qname,
```